pmm/conan.cmake

```diff
@@ -618,9 +612,25 @@ macro(_pmm_conan_install)
         set(__conan_inc "${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake")
         _pmm_log("We are being built by Conan, so we won't run the install step.")
         _pmm_log("Assuming ${__conan_inc} is present.")
-    else()
-        _pmm_conan_install_1()
-    endif()
+    else ()
+        if (CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE AND NOT ARGUMENTS_BUILD_TYPE AND PMM_CMAKE_MULTI)
```

Comment:
Where does `ARGUMENTS_BUILD_TYPE` come from?

Reply:
It was something leftover from the official Conan wrapper 
https://github.com/conan-io/cmake-conan/blob/develop/conan.cmake#L479